proposals/lip-0042.md

```diff
@@ -134,11 +173,19 @@ The default reward of the block as a 64-bit unsigned integer.
 
 #### After Block Execution
 
-After a block `b` is executed, the `mint` function exposed by the [token module][research:token-module] to assign the reward to the block generator as:
+After a block `b` is executed, the `mint` function exposed by the [Token module][token-module] to assign the reward to the block generator as:
 
 ```python
-blockReward = getBlockReward(b.header)
+(blockReward, reduction) = getBlockReward(b.header)
 mint(b.header.generatorAddress, TOKEN_ID_REWARD, blockReward)
+
+eventDataBytes = encode(
+schema = rewardMintedDataSchema,
```

Comment:
For readability, I would recommend some indentation here. You can even use `autopep8` for having automatic Python formatting.

Reply:
I think, my editor is not recognizing the code snippet in markdown as python, but I did the indentation manually.